megcruzen/browserify_productOrdering
# Added single product to DOM

# Description
Added single (hardcoded) product to DOM to ensure that product HTML works.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout build-product-html`
3. Run server/grunt
4. Verify that a single product is being printed to the DOM.
